test/src/htmlfilesetserv/test/HTMLFileSetServServerTest.java

```diff
@@ -979,17 +961,7 @@ public void testFailReportIndexEmptyLinks() throws Exception {
 		testFail(path, TOKEN1.getToken(), 404,
 				"This KBase report does not contain html links", "cookie");
 	}
-	
-	@Test
-	public void testFailReportNoShockFile() throws Exception {
```

Comment:
Hmm, this is a bummer because I assume it's possible that some of the files that were transferred over from shock to blobstore might be 0 length files, but I can't think of a way to test this without doing some hacky crap that's probably worse than not having the test. Oh well. If you have any ideas though...